airgun/views/computeresource.py

```diff
@@ -108,10 +108,30 @@ class RHVProviderForm(View):
         password = TextInput(id='compute_resource_password')
         api4 = Checkbox(id='compute_resource_use_v4')
         load_datacenters = Text("//*[contains(@id,'test_connection_button')]")
+        datacenter = FilteredDropdown(id='s2id_compute_resource_uuid')
         certification_authorities = TextInput(id='compute_resource_public_key')
 
+        # some of the widgets values need to be filled after the main values
```

Comment:
```
url = TextInput(id='compute_resource_url')
user = TextInput(id='compute_resource_user')
password = TextInput(id='compute_resource_password')
api4 = Checkbox(id='compute_resource_use_v4')
certification_authorities = TextInput(id='compute_resource_public_key')
@View.nested
class datacenter(View):
    load_datacenters = Text("//a[contains(@id,'test_connection_button')]")
    datacenter_value = FilteredDropdown(id='s2id_compute_resource_uuid')

    def before_fill(self, values=None):
        self.load_datacenters.click()
```
then
`provider_content.datacenter.datacenter_value: 'value'`
 
All widgets are filled in strict order, one by one, so `datacenter` view will be filled last

Reply:
@oshtaier thanks, this is a more simple implementation